jsk_panda_robot/jsk_panda_startup/env-hooks/99.jsk_panda_startup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+rossetdualpanda() {
+    rossetmaster dual_panda.jsk.imi.i.u-tokyo.ac.jp
+    rossetip
+}
+
+rossetdualpanda2() {
```

Comment:
what is the difference between dualpanda and dualpanda2 ? is that 2nd setup of two-armed panda robot? what's happens if we purchase another 2-set of panda robot?
If you want to use rossetdualpanda, you should  add them on your own .bashrc file, add add README to how to write settings.

Reply:
I followed PR2 style
https://github.com/jsk-ros-pkg/jsk_robot/blob/b40df2f1086bfd8b0c76a83284a7a3a5ad9b85f6/jsk_pr2_robot/jsk_pr2_startup/env-hooks/99.jsk_pr2_startup.bash#L4-L14
and HRP2 style
https://github.com/start-jsk/rtmros_hrp2/blob/master/jsk_hrp2_ros_bridge/env-hooks/99.jsk_hrp2_ros_bridge.bash
where robots have different `rosset*` even if they are the same robot except IP address.
So,
> if we purchase another 2-set of panda robot

`rossetdualpanda3` is created.
Do you mean PR2 style and HRP2 style are wrong?
If so, OK, I'll delete `rossetdualpanda*` and rewrite README.

Reply:
> Do you mean PR2 style and HRP2 style are wrong?

I wouldn't say that's wrong. But using  `rossetip <master_ip>; rossetip` seems simpler than ,some robot uses `rosset<robot>` but some robot uses `rossetmaster + rossetip`. You can leave PR2/HRP2, they have historical reason + need extra environment variables

Reply:
OK, I'll delete `rossetdualpanda*`

Reply:
Done